src/utils/CrossChainBridge.ts

```diff
@@ -160,11 +180,14 @@ export const handleFederatorVoted = (event: VotedEvent, transaction: Transaction
   // TODO: if token is native to RSK, then symbol should be from token entity and not side token
   crossTransfer.symbol = event.params.symbol
   crossTransfer.sender = event.params.sender
-  crossTransfer.votes = crossTransfer.votes + 1
+  if (isSignatureFederation(event.address.toHex().toLowerCase())) {
```

Comment:
This logic is repeated, code can be dried

Reply:
true, however doing it this way keeps the logic of updating the federation entity separate from the logic of updating the crossTranfer entity
for a cleaner code, I will refactor the code to two different methods